tests/actions/settingEncounterRole.test.js

```diff
@@ -45,8 +46,33 @@ describe ('Get the encounterRole configuration actions', () => {
 
             const expectedActions = [
                 SETTING_ENCOUNTER_ROLE_LOADING,
+                SETTING_ENCOUNTER_ROLE_SUCCESS,
+                FETCH_ENCOUNTER_ROLE_LOADING,
+                SETTING_ENCOUNTER_ROLE_LOADING,
+            ];
+            
+            const store = mockStore({ setting: {}});
+
+            return store.dispatch(getSettingEncounterRole()).then(() => {
+                const dispatchedActions = store.getActions();
+                const dispatchedActionTypes = dispatchedActions.map(action => action.type);
+                expect(dispatchedActionTypes).toEqual(expectedActions);
+            });
+        });
+
+        it ('should dispatch `SETTING_ENCOUNTER_ROLE_SUCCESS` with empty results', () => {
```

Comment:
@larrystone, I dont see `SETTING_ENCOUNTER_ROLE_SUCCESS` in the expected actions.

Reply:
Oh, my bad, the implementation logic is that it should throw a failure if an empty result was returned when fetching SETTING_ENCOUNTER_ROLE. I will update the test case description accordingly. Thanks